packages/frontend/scss/message/_message-attachment.scss

```diff
@@ -209,3 +209,8 @@
     }
   }
 }
+
+// no border radius for stickers
+.type-sticker .message-attachment-media {
+  border-radius: 0px;
+}
```

Comment:
We already have a place with this selector

https://github.com/deltachat/deltachat-desktop/blob/ff6b346ed393c28b3d73a562bf7e3ba9807ea72a/packages/frontend/scss/message/_message.scss#L299

Would be nice not to spread related code over multiple files.